src/v-tooltip.js

```diff
@@ -192,7 +197,9 @@ class SuperTooltip extends Tooltip {
 
     // Fix position
     setTimeout(() => {
-      this.popperInstance.update()
+      if (this.popperInstance) {
+        this.popperInstance.update()
```

Comment:
Also an error here appeared once to me, something like `Cannot read property 'update' of undefined`